api/v2/helpers/bouquets/bouquets_helper.py

```diff
@@ -9,3 +13,87 @@ def query_all_bouquets():
         bouquets = BouquetsSchema(many=True).dump(all_bouquets)
 
     return bouquets
+
+
+def query_bouquet(bouquet_id):
+    bouquet_query = BouquetsModel.query.filter_by(
+        state='active', id=bouquet_id).first()
+    bouquet = BouquetsSchema().dump(bouquet_query)
+    return bouquet
+
+
+def refresh_bouquet_channels(api_type, bouquet_id):
+    """helper function for refreshing channels"""
+    response = {'response': {'Error': 'Bouquet not found!'}, 'code': 404}
+    bouquet = query_bouquet(bouquet_id)
+    if bouquet:
+        response = {'response':
+                    {"Error": "Refresh endpoint not responding appropriatly"},
+                    'code': 424
```

Comment:
This is to check if the refresh URL endpoint is responding as required. If an error occurs at this point it is to be because of that dependency and not necessarily on our endpoint